### PR TITLE
Extract command runner and update readme

### PR DESCRIPTION
**Pull request by minhqdao**
Update readme so people know how to use it. 😉

### CODE REVIEW

**Review by jxstxn1 — APPROVED** [review]
LGTM

**Review by passsy — CHANGES_REQUESTED** [review]
That's not how it's supposed to work. Let me adjust it

File: sidekick/lib/src/init/sidekick_command_runner.dart [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import 'package:args/command_runner.dart';
+
+class SidekickCommandRunner extends CommandRunner {
```
Comment (passsy) [inline sidekick/lib/src/init/sidekick_command_runner.dart:3]: > Extract sidekick_command_runner.dart so it is more easily found.

Why would you ever search for it?

Comment (passsy) [reply sidekick/lib/src/init/sidekick_command_runner.dart:3]: I'd prefer to make it private instead and leave it in the main file

Comment (minhqdao) [reply sidekick/lib/src/init/sidekick_command_runner.dart:3]: > > Extract sidekick_command_runner.dart so it is more easily found.
> 
> Why would you ever search for it?

It was when I still thought, I'd need to manually import it

**Comment by minhqdao** [comment]
> That's not how it's supposed to work. Let me adjust it

Then we shouldn't mention it that way under `Creating sub commands` but instead point out that that's being generated by tool so you don't have to write it yourself

**Comment by minhqdao** [comment]
I was just working on it :(

**Pull request merged**